sdk/src/logs/simple_log_processor.cc

```diff
@@ -43,6 +50,7 @@ void SimpleLogProcessor::OnReceive(std::unique_ptr<Recordable> &&record) noexcep
       /* Alert user of the failed export */
     }
   }
+#  ifdef ENABLE_ASYNC_EXPORT
```

Comment:
Just like in `SimpleSpanProcessor`, `if (is_export_async_ == false)` should only be checked when `ENABLE_ASYNC_EXPORT` is defined, or we should always call sync Export.

Reply:
Done